_docs-2-0/administration/caching.md

```diff
@@ -31,7 +31,8 @@ setting {% plink table.cache.block.enable %} to `true` in the shell:
 Or programmatically using [TableOperations.setProperty()][tableops]:
 
 ```java
-conn.tableOperations().setProperty("mytable", "table.cache.block.enable", "true");
+AccumuloClient client = ... ;
```

Comment:
I don't think you need this.  The name "client" is even more obvious than "conn".

Reply:
I agree. I'll remove it from any advanced documentation.